### PR TITLE
Generate new APIs for PHP

### PR DESCRIPTION
**Pull request by michaelbausor**
- ErrorReporting
- Speech
- Vision
- Monitoring
- Trace
- Language

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
LGTM - I only have requests to file issues, and a comment that can be addressed in a follow up.

**Comment by michaelbausor** [comment]
Re this: I don't see any review comments. Did you leave comments here about issues to file?

File: generated/php/google-cloud-speech-v1beta1/src/Speech/V1beta1/SpeechApi.php [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+    private $defaultCallSettings;
+    private $descriptors;
+
+    private static function getPageStreamingDescriptors()
+    {
+        $pageStreamingDescriptors = [
+        ];
+
+        return $pageStreamingDescriptors;
+    }
```
Comment (michaelbausor) [inline generated/php/google-cloud-speech-v1beta1/src/Speech/V1beta1/SpeechApi.php]: https://github.com/googleapis/toolkit/issues/560

File: generated/php/google-devtools-clouderrorreporting-v1beta1/src/Errorreporting/V1beta1/ReportErrorsServiceApi.php [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+    private static function getPageStreamingDescriptors()
+    {
+        $pageStreamingDescriptors = [
+        ];
+
+        return $pageStreamingDescriptors;
+    }
+
+    // TODO(garrettjones): add channel (when supported in gRPC)
```
Comment (michaelbausor) [inline generated/php/google-devtools-clouderrorreporting-v1beta1/src/Errorreporting/V1beta1/ReportErrorsServiceApi.php]: https://github.com/googleapis/toolkit/issues/561

File: generated/php/google-devtools-cloudtrace-v1/src/Trace/V1/TraceServiceApi.php [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+            'getTrace' => $defaultDescriptors,
+            'listTraces' => $defaultDescriptors,
+        ];
+        $pageStreamingDescriptors = self::getPageStreamingDescriptors();
+        foreach ($pageStreamingDescriptors as $method => $pageStreamingDescriptor) {
+            $this->descriptors[$method]['pageStreamingDescriptor'] = $pageStreamingDescriptor;
+        }
+
+        // TODO load the client config in a more package-friendly way
+        // https://github.com/googleapis/toolkit/issues/332
```
Comment (michaelbausor) [inline generated/php/google-devtools-cloudtrace-v1/src/Trace/V1/TraceServiceApi.php]: Yes, will remove

**Pull request merged**